### PR TITLE
Add information about model

### PR DESCRIPTION
**Pull request by EgorBu**
* training parameters
* a number of rules & avg. len
* load model before features to make it fail fast in case of problems with a model
Signed-off-by: egor <egor@sourced.tech>

### CODE REVIEW

File: lookout/style/format/quality_report.py [hunk 1]
```diff
@@ -46,15 +46,18 @@ def prepare_files(folder: str, client: BblfshClient, language: str) -> Iterable[
 def quality_report(input_pattern: str, bblfsh: str, language: str, n_files: int, model: str
                    ) -> None:
     """Print several different reports for a given model on a given dataset."""
+    analyzer = FormatModel().load(model)
+    rules = analyzer._rules_by_lang[language]
+    print("Model parameters: %s" % rules.origin)
+    print("Number of rules: %s, avr. len.: %.1f" % (len(rules.rules), rules.avg_rule_len))
```
Comment (vmarkovtsev) [inline lookout/style/format/quality_report.py]: This is printed by `str(rules)` - I wrote about it :)
```python
print(rules)
```

File: lookout/style/format/quality_report.py [hunk 2]
```diff
@@ -46,15 +46,18 @@ def prepare_files(folder: str, client: BblfshClient, language: str) -> Iterable[
 def quality_report(input_pattern: str, bblfsh: str, language: str, n_files: int, model: str
                    ) -> None:
     """Print several different reports for a given model on a given dataset."""
+    analyzer = FormatModel().load(model)
+    rules = analyzer._rules_by_lang[language]
```
Comment (vmarkovtsev) [inline lookout/style/format/quality_report.py]: Sorry, missed this in the previous review.

```python
rules = analyzer[language]
```

Besides, please rename `analyzer` to `model` and `model` to `model_path`. We never load analyzers, very confusing.

**Comment by vmarkovtsev** [comment]
@EgorBu ping

File: lookout/style/format/visualization.py [hunk 3]
```diff
@@ -70,9 +73,9 @@ def visualize(input_filename: str, bblfsh: str, language: str, model: str) -> No
 
         new_content += GREEN + CLASSES[wrong.y] + RED + CLASSES[wrong.pred] + ENDC
 
-        if i == len(mispred) - 1 and end != len(old_content):
-            new_content += old_content[end:]
+        if i == len(mispred) - 1:
+            if end != len(old_content):
```
Comment (vmarkovtsev) [inline lookout/style/format/visualization.py]: @m09 fyi - tricky fix here

Comment (vmarkovtsev) [reply lookout/style/format/visualization.py]: BTW @EgorBu Would be nice to put a comment here

Comment (EgorBu) [reply lookout/style/format/visualization.py]: will do in one of the next PRs

**Comment by m09** [comment]
Didn't run to check but I think this PR broke `__main__.py` since model arg name stayed the same and is now `model_path` in both files

**Comment by vmarkovtsev** [comment]
@m09 Why didn't we catch this in the integration test :thinking:

**Comment by EgorBu** [comment]
https://github.com/src-d/style-analyzer/pull/155 - this PR should fix it - locally it works

**Pull request merged**